cypress/e2e/Pages/LoginPage.js

```diff
@@ -0,0 +1,32 @@
+class loginPage {
+    visit() {
+      cy.visit('https://www.saucedemo.com/');
```

Comment:
Please use BaseURL in the config file.
https://docs.cypress.io/guides/references/configuration